exa/core/box.py

```diff
@@ -31,30 +45,96 @@ def groupby(self):
         pass
 
     def info(self):
-        pass
+        info = []
+        for name, data in self._data.items():
+            df = data.data()
+            # TODO: prepend __module__?
```

Comment:
Does this mean make the "type" column look like `pandas.DataFrame`, etc.?

Reply:
I suppose the type(df) doesn't quite make it in the translation. Will have to update this.